main.py

```diff
@@ -83,5 +136,5 @@ def poll_status(job_id = None):
 
 
 if __name__ == '__main__':
-    send_media()
-
+    #transcribe_and_redact()
```

Comment:
Remove parens

Reply:
There are two different functions. One for transcription and one for redaction